movie-project/src/components/MoviesGrid.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import MovieItem from "./MovieItem";
+
+export default function MoviesGrid(props) {
```

Comment:
extract props here too like 
```suggestion
export default function MoviesGrid({ isLoaded,  setIsLoaded, genresList }) {
```